pallets/messages/src/rpc/src/lib.rs

```diff
@@ -34,25 +42,89 @@ impl<C, M> MessagesHandler<C, M> {
 	}
 }
 
+#[derive(Debug)]
+pub enum MessageRpcError {
+	/// Pagination request is bad
+	InvalidPaginationRequest,
+	/// Likely passed u32 block count
+	TypeConversionOverflow,
+	/// Schema Id doesn't exist or errored when retrieving from state
+	InvalidSchemaId,
+}
+
+impl From<MessageRpcError> for RpcError {
+	fn from(e: MessageRpcError) -> Self {
+		RpcError::Custom(format!("{:?}", e))
```

Comment:
Maps the error enum onto "custom" RPC errors.

These could be a lot better, but effectively matches what existed before.